elyra/pipeline/tests/resources/sample_pipelines/pipeline_with_custom_components.json

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
Should probably rename this file to indicate that the custom components are Airflow components